src/assets/stylesheets/index.scss

```diff
@@ -356,3 +356,45 @@ body {
   }
 }
 
+#mc_embed_signup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+#mc_embed_signup_scroll {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+:local(.subscribeBar) {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+:local(.subscribe-text) {
+  font-size: 14pt;
+  font-weight: lighter;
+  margin: 0px 0px 8px 0px;
+}
+
+:local(.subscribe-button) {
+  @extend %action-button-secondary;
+  height: 40px;
+}
+
+:local(.email-input) {
+  height: 40px;
+  border-radius: 24px;
+  margin: 0px 16px 0px 0px;
+  width: 24em;
+  text-align: center;
+}
+
+.mc_field-group {
```

Comment:
Does this have to be a bare classname instead of a `:local` class?

Reply:
No, good point.